dist-material/release-docs/LICENSE

```diff
@@ -327,6 +327,7 @@ The text of each license is the standard Apache 2.0 license.
     Armeria 1.11.0, http://github.com/line/armeria, Apache 2.0
     Brotli4j 1.6.0, https://github.com/hyperxpro/Brotli4j, Apache 2.0
     micrometer 1.7.3, https://github.com/micrometer-metrics/micrometer, Apache 2.0
+    iotdb-session 0.12.2, https://github.com/apache/iotdb, Apache 2.0
```

Comment:
All dependency libs should be added here and https://github.com/apache/skywalking/blob/master/tools/dependencies/known-oap-backend-dependencies.txt.

Reply:
I've modified `known-oap-backend-dependencies.txt` before

Reply:
You just added one line, we check for all real dependencies.

Reply:
Read the error logs, they are much more than 2

![image](https://user-images.githubusercontent.com/5441976/134611759-efc6e815-0a3c-4415-be83-21f3879a8210.png)

Reply:
Got it.